src/components/TimelineNav/TimelineNav.tsx

```diff
@@ -247,7 +250,12 @@ export const TimelineNav = ({
       return child;
```

Comment:
I can't leave a comment up there but can you change the `onKeyDown` method to use `e.key` instead of `e.code`? The keyboard navigation down the list isn't working and I think that's why (this has come up a couple times).

Reply:
Done